electricitymap/contrib/lib/models/events.py

```diff
@@ -178,14 +179,16 @@ def _validate_zone_key(cls, v):
         return v
 
     @validator("datetime")
-    def _validate_datetime(cls, v, values: Dict[str, Any]):
+    def _validate_datetime(cls, v: dt.datetime, values: Dict[str, Any]):
```

Comment:
Since there's an attribute datetime in the object, we need to rename the module a bit for type checker to understand it.